src/book/02-system/01-foundations/01-overview.mdx

```diff
@@ -31,45 +31,39 @@ When adding modules to the list in a role, use the syntax `path::to::module`. Fo
-| `scripttuner`         | Tune/modify/create [scripts](/system/subsystems/motion#script-engine).                                                                                                                                 |
-| `test_sensor`         | Test odometry (needs NUsight). Check for broken servos without the robot moving, because servo errors and warnings will be printed to the terminal.                                                    |
-| `fake_visualmesh`     | Test the vision system, such as detection of balls and goals, in a simulated environment.                                                                                                              |
-| `visualmesh`          | Test the vision system on a real robot.                                                                                                                                                                |
-| `webots_scripttuner`  | Tune/modify/create [scripts](/system/subsystems/motion#script-engine) in the webots simulator.                                                                                                         |
-| `webots_keyboard`     | Test motions in Webots with manual control.                                                                                                                                                            |
-| `webots`              | Test the connection between the robot code and Webots.                                                                                                                                                 |
-| `webots_robocup`      | Play RoboCup games in the Webots simulator.                                                                                                                                                            |
-
-In a role, you should have only one platform. Possible platforms are `platform::darwin::HardwareIO`, `platform::darwin::HardwareSimulator`, `platform::Webots` and `platform::Gazebo`.
+| Role Name               | Usage                                                                                                                                                                                                  |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `fake/visualmesh`       | Test the vision system, such as detection of balls and goals, in a simulated environment.                                                                                                              |
+| `fake/keyboardwalk`     | Test parts of the code without a real robot. Demonstrate basic functionality of the robot. Part of the [Getting Started](/guides/main/getting-started) example since it does not require a real robot. |
```

Comment:
Extremely picky but can we swap `fake/keyboardwalk` and `fake/visualmesh`? Everything else is in alphabetical order except these two